### PR TITLE
Review frontend

### PR DESCRIPTION
**Pull request by Jason-Williams-github**
Sorry about that. had to fix some styling stuff I had that was helping me with the layout

### CODE REVIEW

**Review by StephenDMay — APPROVED** [review]

**Review by YesratRahman — APPROVED** [review]

**Review by wcollins37 — APPROVED** [review]

**Review by tamera-brown — APPROVED** [review]

File: nile-backend/src/test/java/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/NileTest
```
Comment (zmiles17) [inline nile-backend/src/test/java/resources/application.properties:1]: Is this file necessary? I put this information in the application-test.properties.

**Review by zmiles17 — COMMENTED** [review]
Do we need application.properties in the test folder? I added an application-test.properties specifically for testing purposes.

File: nile-backend/src/test/java/resources/application.properties (hunk 1 above)
Comment (Jason-Williams-github) [reply nile-backend/src/test/java/resources/application.properties]: yeah, my bad. Left that in there when I was testing before the one you added was in. I deleted it
